clusterloader2/testing/density/config.yaml

```diff
@@ -15,7 +15,7 @@
 # This results in decreased probability of choosing the same node again.
 {{$LATENCY_POD_CPU := DefaultParam .LATENCY_POD_CPU 100}}
 {{$LATENCY_POD_MEMORY := DefaultParam .LATENCY_POD_MEMORY 350}}
-{{$MIN_LATENCY_PODS := 500}}
+{{$MIN_LATENCY_PODS := MinInt (MultiplyInt $PODS_PER_NODE .Nodes) 500}}
```

Comment:
`PODS_PER_NODE * Nodes` will be greater than 500 in all cases that we currently run; existing tests are therefore unaffected. However, in the requiredDuringScheduling anti-affinity tests, we cannot schedule more than one pod per node, so we need this.